src/components/APRLabel/APRLabel.js

```diff
@@ -22,9 +22,9 @@ import GlpManager from "../../abis/GlpManager.json";
 
 import { useWeb3React } from "@web3-react/core";
 
-import { useGmxPrice } from "../../Api";
+import { useGmxPrice } from "../../domain";
```

Comment:
i've mentioned that in README discussion, maybe better to not use domain/index? maybe move everything from index.js to common.js and import explicitly `from "domain/common"`? not sure